website/integrations/services/fortigate-admin/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: FortiGate Admin Login
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is FortiGate
+
+> FortiGate is a firewall from FortiNet. It is a NGFW with layer7 inspection and able to become a part of a FortiNet security fabric.
+> -- https://www.fortinet.com/products/next-generation-firewall
+>
+> This guide explains how to setup a FortiGate to use authentik as SAML provider for Admin Login. It does not cover how to setup SSLVPN logins, that is a different configuration.
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `fgt.company` is the FQDN of the FortiGate install.
+-   `authentik.company` is the FQDN of the authentik install.
+-   `fgt.mapping` is the name of the SAML Property Mapping.
+-   `ak.cert` = The authentik self-signed certificate you use for the service provider.
+
+> [!IMPORTANT]
+> If you have changed the port of the admin login from 443 to anything else you have to append it behind `fgt.company`. So f.e. `fgt.company:10443`.
+
+## Custom Propery Mapping
+
+Create a new SAML Property Mapping under the Customization settings.
+
+-   `fgt.mapping` is the value for the Name.
+-   `username` is the value for the SAML Attribute Name.
+-   `return request.user.email` is the value for the Expression.
+
+Create an application and Provider in authentik, note the slug, as this will be used later. Create a SAML provider with the following parameters:
```

Comment:
```suggestion
Create an application and SAML provider in authentik, and note the slug, because this will be used later. Create a SAML provider with the following parameters:
```